cmd/soroban-rpc/lib/preflight/src/fees.rs

```diff
@@ -157,54 +161,43 @@ fn calculate_host_function_soroban_resources(
 fn get_fee_configurations(
     ledger_storage: &ledger_storage::LedgerStorage,
     bucket_list_size: u64,
-) -> Result<(FeeConfiguration, RentFeeConfiguration), Box<dyn error::Error>> {
+) -> Result<(FeeConfiguration, RentFeeConfiguration)> {
     let ConfigSettingEntry::ContractComputeV0(compute) =
         ledger_storage.get_configuration_setting(ConfigSettingId::ContractComputeV0)?
     else {
-        return Err(
-            "get_fee_configuration(): unexpected config setting entry for ComputeV0 key".into(),
-        );
+        bail!("unexpected config setting entry for ComputeV0 key");
+
     };
 
     let ConfigSettingEntry::ContractLedgerCostV0(ledger_cost) =
         ledger_storage.get_configuration_setting(ConfigSettingId::ContractLedgerCostV0)?
     else {
-        return Err(
-            "get_fee_configuration(): unexpected config setting entry for LedgerCostV0 key".into(),
-        );
+        bail!(
+           "unexpected config setting entry for LedgerCostV0 key");
```

Comment:
Nit: Is this actually rustfmt? Looks weird.

Reply:
Uhm, it is happy either way. I moved everything to the same line.